cmake/onnxruntime_kernel_explorer.cmake

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+if(CMAKE_CXX_COMPILER MATCHES ".*hipcc$")
+  message(FATAL_ERROR "don't use hipcc!")
+endif()
+
+if(NOT onnxruntime_ENABLE_PYTHON)
+  message(FATAL_ERROR "python is required but is not enabled")
+endif()
+
+if(NOT HIP_FOUND)
+  message(FATAL_ERROR "hip is required but is not found")
+endif()
+
+set(KERNEL_EXPLORER_ROOT ${ONNXRUNTIME_ROOT}/python/tools/kernel_explorer)
+set(BERT_DIR ${ONNXRUNTIME_ROOT}/contrib_ops/rocm/bert)
+
+file(GLOB kernel_explorer_srcs CONFIGURE_DEPENDS "${KERNEL_EXPLORER_ROOT}/*.cc")
+file(GLOB kernel_explorer_kernel_srcs CONFIGURE_DEPENDS "${KERNEL_EXPLORER_ROOT}/kernels/*.cc")
+
+onnxruntime_add_shared_library_module(kernel_explorer
+  ${kernel_explorer_srcs}
+  ${kernel_explorer_kernel_srcs}
+  ${BERT_DIR}/util.cc)
```

Comment:
Do we need util.cc? 
As util is already part of ORT, and we depend on ORT, maybe we don't need it include twice here?

Reply:
This will require us to enable contrib ops build, which will causing _ZTVN11onnxruntime7contrib4ATenE. That is, we need a whole bunch of pytorch libraries. Since we only depends on a small bunch of utility functions in BERT_DIR, this will not add much complexity, so let's keep it as is, at least for now.

Reply:
In ORT we actually don't include from `.`,, kernel explorer is an exception.